backend/blueprints/spa_api/service_layers/replay/replay_positions.py

```diff
@@ -62,3 +69,7 @@ def process_player_df(game) -> List[ReplayPlayer]:
     @staticmethod
     def filter_frames(frames, data_frame):
         return data_frame.loc[data_frame.index.isin(frames)]
+
+    @staticmethod
+    def filter_frames_range(fmin, count, data_frame):
+        return data_frame.loc[(data_frame.index >= fmin) & (data_frame.index < fmin + count)]
```

Comment:
Do you know if there is a performance difference between having the addition in the quiet and having it before?

Reply:
No performance difference. It's just a like a loop, it's evaluated before any comparison is done (and way before any indexing)